lib/parallel_tests/cli.rb

```diff
@@ -44,6 +44,7 @@ def execute_in_parallel(items, num_processes, options)
           simulate_output_for_ci options[:serialize_stdout] do
             Parallel.map(items, :in_threads => num_processes) do |item|
               result = yield(item)
+              ParallelTests.stop_all_processes if result[:exit_status] != 0 && options[:fail_fast]
               reprint_output(result, lock.path) if options[:serialize_stdout]
```

Comment:
- should go after the printing ?
- parallel has `raise Parallel::Kill`, that should work too ?

Reply:
- I will change the order

- raise the exception doesn't seems to be right solution. I would expect that tests will shutdown softly after running they after hooks, where they could perform cleaning code.